src/vit-testing/vitup/src/mode/standard/controllers/vit_station/data.rs

```diff
@@ -39,12 +33,10 @@ impl DbGenerator {
     ) -> Result<(), Error> {
         std::fs::File::create(db_file)?;
 
-        let migration_scripts_path = MigrationFilesBuilder::default().build(&self.root)?;
         let mut generator = ValidVotePlanGenerator::new(self.parameters);
         let snapshot = generator.build(template_generator);
         DbBuilder::new()
             .with_snapshot(&snapshot)
-            .with_migrations_from(migration_scripts_path)
```

Comment:
Did this so we stop using different migration files in vit-ss and vitup since now migrations are embedded in the code.